dashboard.csss-ci.com/server/api/models/Suppliers_model.php

```diff
@@ -1,16 +1,34 @@
 <?php
 
+/**
+tests to see if BASEPATH is defined
+*/
 if (!defined('BASEPATH'))
     exit('No direct script access allowed');
-
+/**
+class that controls Suppliers model
+extends from CI_Model
+*/
 class Suppliers_model extends CI_Model {
-
+    
+    /**
+    private var for table
+    */
     protected $table = "suppliers";
-
+    
+    /**
+    saves data
+    @param $data seems to be a list or array
```

Comment:
- No seems, if you are unsure, find the answer
- Format should be as follows:
/**
\* [Description of the method]
\*
\* @param [type] [param name] - [description of param]
\* @param [type] [param name] - [description of param]
\* @return [type] [description of return]
*/